modules/virt-configuring-ssh.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-ssh.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-ssh_{context}"]
+= Configuring SSH for quick virtual machine access
+
+You can configure your local SSH client to quickly connect to your virtual machine (VM). This configuration makes it possible to leverage `virtctl` to relay an SSH command to the target VM.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `virtctl` client.
+* The virtual machine you want to access is running.
+
+.Procedure
+
+. Modify the `~/.ssh/config` file on your client machine to include the following text:
++
+[source,terminal]
+----
+Host vmi/*
+  ProxyCommand virtctl port-forward --stdio=true %h %p
+----
+
+. Test the connection by running the following command:
++
+[source,terminal]
+----
+$ ssh <user>@<vmi>.<vminame>.<mynamespace>

```

Comment:
Please change the command to variables or examples. If variables, `<user>@<vm_name>.<namespace>`. If examples, `user@example-vm.example-namespace`. Putting examples in `<>` is against the OCP style guidelines.